docs/features/recce-debug.md

```diff
@@ -0,0 +1,113 @@
+[OK] Directory exists: target-base
+[OK] Manifest JSON file exists : target-base/manifest.json
+[OK] Catalog JSON file exists: target-base/catalog.json
+────────────────────── Warehouse Connection ─────────────────────────
+[OK] Connection test
+──────────────────────────── Result ─────────────────────────────────
+[OK] Ready to launch! Type 'recce server'.
+```
+
+### Partial Setup Cases
+
+Even when some components are missing, Recce can still launch with limited features and provides helpful tips:
+
+#### Missing Base Environment Directory
+
+```
+────────────────────── Development Environment ──────────────────────
+[OK] Directory exists: target
+[OK] Manifest JSON file exists : target/manifest.json
+[OK] Catalog JSON file exists: target/catalog.json
+─────────────────────────── Base Environment ────────────────────────
+[MISS] Directory not found: target-base
+────────────────────── Warehouse Connection ─────────────────────────
+[OK] Connection test
+──────────────────────────── Result ─────────────────────────────────
+[OK] Ready to launch with limited features. Type 'recce server'.
+[TIP] Run dbt with '--target-path target-base' or overwrite the
+      default directory of the base environment with
+      '--target-base-path'.
+```
+
+#### Missing Base Environment Artifacts
+
+```
+────────────────────── Development Environment ──────────────────────
+[OK] Directory exists: target
+[OK] Manifest JSON file exists : target/manifest.json
+[OK] Catalog JSON file exists: target/catalog.json
+─────────────────────────── Base Environment ────────────────────────
+[OK] Directory exists: target-base
+[MISS] Manifest JSON file not found: target-base/manifest.json
+[MISS] Catalog JSON file not found: target-base/catalog.json
+────────────────────── Warehouse Connection ─────────────────────────
+[OK] Connection test
+──────────────────────────── Result ─────────────────────────────────
+[OK] Ready to launch with limited features. Type 'recce server'.
+[TIP] 'dbt run --target-path target-base' to generate the
+      manifest JSON file for the base environment.
+[TIP] 'dbt docs generate --target-path target-base' to generate
+      the catalog JSON file for the base environment.
+```
+
+#### Connection Failure
+
+When the database connection fails, you'll see output like this:
+
+```
+────────────────────── Development Environment ──────────────────────
+[OK] Directory exists: target
+[OK] Manifest JSON file exists : target/manifest.json
+[OK] Catalog JSON file exists: target/catalog.json
+─────────────────────────── Base Environment ────────────────────────
+[OK] Directory exists: target-base
+[OK] Manifest JSON file exists : target-base/manifest.json
+[OK] Catalog JSON file exists: target-base/catalog.json
+────────────────────── Warehouse Connection ─────────────────────────
+[FAIL] Connection test
+──────────────────────────── Result ─────────────────────────────────
+[TIP] Run 'dbt debug' to check the connection.
```

Comment:
So if the connection failure, users cannot run Recce? 
the tip here feels like starting an infinite loop.

Reply:
It's 'dbt debug' not 'recce debug'. I think it's fine?